### DISCUSSION TITLE
Znikneły encje

### DISCUSSION THREAD

**Discussion by Wilu88** [PiotrMachowski/Home-Assistant-custom-components-Burze.dzis.net#18] [Q&A]
hej. Nie wiem czy to problem mojego HA czy coś nid tak z API, ale nagle dzisiaj znikneły encje: burze_dzis_net_wind_warning, burze_dzis_net_precipitation_warning, burze_dzis_net_storm_warning, burze_dzis_net_storms_nearby.

**Comment by PiotrMachowski** [comment]
Też nie wiem, u mnie działa

**Reply by PiotrMachowski** [reply]
Coś w logach? Cokolwiek zmieniłeś w HA?

**Reply by Wilu88** [reply]
Zmieniać nic nei zmieniałem,standardowo tylko update HA. Ale skoro u Ciebie działą to usuwam i dodaje jeszcze raz ;) Myślałem ze moze API się wysypało.
